### PR TITLE
Add tests for UpdateMeetingDescriptor

### PR DESCRIPTION
**Pull request by muraddurrani**
- Add tests for UpdateMeetingDescriptor
- Change the setAttendees and getAttendees methods in updateMeetingDescriptor to setIndexes and getIndexes for consistency, since in Meeting the attendees refers to a `Set<Id>` and indexes refers to a `Set<Index>`

### CODE REVIEW

File: src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java [hunk 1]
```diff
@@ -30,33 +35,45 @@ public UpdateMeetingDescriptorBuilder(Meeting meeting) {
         descriptor.setAgenda(meeting.getAgenda());
         descriptor.setMeetingTime(meeting.getTime());
         descriptor.setMeetingPlace(meeting.getPlace());
-        descriptor.setAttendees(meeting.getIndexes());
+        descriptor.setIndexes(meeting.getIndexes());
     }
 
     /**
-     * Sets the {@code Agenda} of the {@code EditPMeetingDescriptor} that we are building.
+     * Sets the {@code Agenda} of the {@code UpdatePMeetingDescriptor} that we are building.
```
Comment (NicsunXnus) [inline src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java:42]: typo?

File: src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java [hunk 2]
```diff
@@ -30,33 +35,45 @@ public UpdateMeetingDescriptorBuilder(Meeting meeting) {
...
      */
     public UpdateMeetingDescriptorBuilder withAgenda(String agenda) {
         descriptor.setAgenda(new Agenda(agenda));
         return this;
     }
 
     /**
-     * Sets the {@code MeetingTime} of the {@code EditPMeetingDescriptor} that we are building.
+     * Sets the {@code MeetingTime} of the {@code UpdatePMeetingDescriptor} that we are building.
```
Comment (NicsunXnus) [inline src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java:50]: Looks like they are all the same

File: src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java (hunk 1 above)
Comment (muraddurrani) [reply src/test/java/seedu/address/testutil/UpdateMeetingDescriptorBuilder.java:42]: Ah sorry will fix one sec

**Review by NicsunXnus — APPROVED** [review]
LGTM,merging

**Pull request merged**